testing/testcontainers/java/src/test/java/io/crate/example/testing/TestClassScope.java

```diff
@@ -29,12 +32,11 @@ public void testReadSummits() throws SQLException, IOException {
 
         // Get JDBC URL to CrateDB instance.
         String connectionUrl = cratedb.getJdbcUrl();
-        System.out.println(String.format("Connecting to %s", connectionUrl));
+        System.out.printf("Connecting to %s%n", connectionUrl);
```

Comment:
Sweet! Why do I use `%s%n` for interpolating a **single** value?

Reply:
%n is to print the os new line, so `\r\n` for win, `\n` for Unix/Linux/MacOs

Reply:
You can also use: `System.out.println("Connection url: " + url)`

Reply:
All right. Thanks!